test/integration/manifests/cns/daemonset-windows.yaml

```diff
@@ -0,0 +1,149 @@
+
```

Comment:
nit: the linux daemonset within this directory does not have a configmap section, hence the cnsconfig directory. Is there any reason not to keep that pattern?